book/src/leader-validator-transition.md

```diff
@@ -0,0 +1,104 @@
+# Leader-to-Validator Transition
+
+A fullnode typically operates as a validator. If, however, a staker delegates
+its stake to a fullnode, it will occasionally be selected as a *slot leader*.
+As a slot leader, the fullnode is responsible for producing blocks during an
+assigned *slot*. A slot has a duration of some number of preconfigured *ticks*.
+The duration of those ticks are estimated with a *PoH Recorder* described later
+in this document.
+
+## BankFork
```

Comment:
I hate this name. This thing should be called Bank.  The thing that's currently called Bank could be renamed to `BitrottingBankYouShouldNotUse`.

Reply:
Can we call the global Env :)